From 85c47d704eb5f85c48c9588db054f2639cbce6d3 Mon Sep 17 00:00:00 2001
From: Ben Greiner <code@bnavigator.de>
Date: Wed, 15 Jun 2022 13:14:08 +0200
Subject: [PATCH] Changed clang2py -h output in Python 3.10

The  line for the argparser changed from "optional arguments" to "options".
---
 test/test_clang2py.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test_clang2py.py b/test/test_clang2py.py
index 6cf2ea6..3d0ad43 100644
--- a/test/test_clang2py.py
+++ b/test/test_clang2py.py
@@ -96,7 +96,7 @@ def test_helper(self):
         self.assertEqual(0, p.returncode)
         self.assertIn("Cross-architecture:", output)
         self.assertIn("usage:", output)
-        self.assertIn("optional arguments", output)
+        self.assertRegex(output, r"\noption.*:")
 
 
 class ArgumentTypeKind(ClangTest):
